patches/minecraft/net/minecraft/client/renderer/texture/Texture.java.patch

```diff
@@ -1,12 +1,21 @@
 --- a/net/minecraft/client/renderer/texture/Texture.java
 +++ b/net/minecraft/client/renderer/texture/Texture.java
-@@ -33,6 +33,11 @@
+@@ -33,6 +33,20 @@
        GlStateManager.func_227677_b_(3553, 10240, j);
     }
  
 +   // FORGE: This seems to have been stripped out, but we need it
++   boolean lastBlur;
++   boolean lastMipmap;
```

Comment:
I guess these should be either private or protected, but not package-private, or am I missing something?